src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const reducer = combineReducers({
+  booksReducer,
+  categoriesReducer,
+});
+
+const store = createStore(reducer);
+
+export default store;
```

Comment:
[OPTIONAL] I would recommend Importing `applyMiddleware` to use some handy Redux middleware, like a redux logger. It will help you a lot in the development process. To install it, you needs to run:
- `npm i --save redux-logger `

Reply:
> 

Thank you @medaminedev66  for approving my project. Count on me to add your suggestion to the next branch.